R/ranges_helpers.R

```diff
@@ -143,35 +143,103 @@ asTX <- function(grl, reference) {
   return(pmapToTranscripts(grl, reference))
 }
 
-#' Faster more secure version of mapFromTranscripts
-#'
-#' Fixes a bug in function, and should have 10x speedup
-#' Also removes hit column for you
+#' Faster pmapFromTranscript
 #'
-#' @param ranges IRanges of ranges within grl
-#' @param grl the "transcripts" that contain ranges, GRangesList
-#' @param indices integer vector of which index of grl ranges are from:
-#' (c(1,1,2)) means first two ranges are from grl[1], third from grl[2])
-#' @return A GrangesList of ranges mapped from transcripts, names are
-#'  re-used from the grl (the transcripts).
-#' @family ExtendGenomicRanges
+#' This version tries to fix the shortcommings of GenomicFeature's version.
+#' Much faster and uses less memory.
+#' Implemented as dynamic program optimized c++ code.
```

Comment:
say that its always keeps the names, if no names than it makes them for the user